### PR TITLE
add pymonetdb recipe

### PR DESCRIPTION
**Pull request by dfroger**
Native MonetDB client Python API

### CODE REVIEW

File: recipes/pymonetdb/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    - python
+    - six
+
+test:
+  imports:
+    - pymonetdb
+    - pymonetdb.control
+
+about:
+  home: http://www.monetdb.org
```
Comment (synapticarbors) [inline recipes/pymonetdb/meta.yaml]: It's not clear if this python client is officially supported by monetdb, so I'm not sure if this should be the home of this package. If it isn't, I would just make this either the github url or the docs.

File: recipes/pymonetdb/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  license: MPL-2.0
+  license_family: NONE
```
Comment (synapticarbors) [inline recipes/pymonetdb/meta.yaml]: You can probably just drop the `license_family` here.

File: recipes/pymonetdb/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  license_file: '{{ environ["RECIPE_DIR"] }}/COPYING'
```
Comment (jakirkham) [inline recipes/pymonetdb/meta.yaml]: Please raise an issue upstream to include the license file in the `MANIFEST.in` for future packaging.

**Comment by synapticarbors** [comment]
@dfroger Can you take a quick look at the minor reviews here and make any necessary change. I'm happy to merge it afterward since the PR has been languishing.

**Comment by scopatz** [comment]
pinging this!

File: recipes/pymonetdb/meta.yaml (hunk 1 above)
Comment (dfroger) [reply recipes/pymonetdb/meta.yaml]: Seems this is the official python client:
https://github.com/MonetDB/MonetDB/blob/31d3c91730c94edd212a932ffb75b8cdad40f767/debian/changelog#L584-L586
http://pymonetdb.readthedocs.io/en/latest/introduction.html#introduction
https://www.monetdb.org/Documentation/Manuals/SQLreference/Programming/Python

File: recipes/pymonetdb/meta.yaml (hunk 3 above)
Comment (dfroger) [reply recipes/pymonetdb/meta.yaml]: [done](https://github.com/gijzelaerr/pymonetdb/issues/25)

File: recipes/pymonetdb/meta.yaml (hunk 2 above)
Comment (dfroger) [reply recipes/pymonetdb/meta.yaml]: done

**Comment by dfroger** [comment]
Done, sorry for the delay... Not sure reply quickly on issue/PR, if someone else want to be added to the maintainer list, that's will be great!

**Comment by dfroger** [comment]
I'll try to rerender to fix ci tests failed

**Pull request closed without merging**